sdk/python/foundation-models/system/inference/text-to-image/safe-text-to-image-inpainting-online-endpoint.ipynb

```diff
@@ -0,0 +1,710 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create an Azure AI Content Safety __(AACS)__ enabled Inpainting online endpoint (Preview)\n",
+    "### This notebook is under preview.\n",
+    "\n",
+    "### Steps to create an __AACS__ enabled __inpainting__ online endpoint\n",
+    "1. Create an __AACS__ resource for moderating the request from user and response from the __inpainting__ online endpoint.\n",
+    "2. Create a new __AACS__ enabled __inpainting__ online endpoint with a custom [score_online.py](./aacs-scoring-files/score/score_online.py) which will integrate with the __AACS__ resource to moderate the response from the __text-to-image__ models and the request from the user, but to make the custom [score_online.py](./aacs-scoring-files/score/score_online.py) to successfully authenticated to the __AACS__ resource, we have 2 options:\n",
```

Comment:
text-to-image-inpainting instead of text-to-image

Reply:
No, we don't have a new task type filter for inpainting model. Similar to HuggingFace, models that can perform the `inpainting` task are tagged with `text-to-image`.